packages/gafl-webapp-service/src/pages/order-complete/route.js

```diff
@@ -32,19 +33,21 @@ const getData = async request => {
   const endTimeString = displayEndTime(permission)
 
   return {
+    permission,
+    startTimeString,
+    endTimeString,
+    disabled: permission.concessions ? permission.concessions.find(c => c.type === mappings.CONCESSION.DISABLED) : null,
+    licenceTypes: mappings.LICENCE_TYPE,
+    hasJunior: !!concessionHelper.hasJunior(permission),
+    hasSenior: !!concessionHelper.hasSenior(permission),
```

Comment:
Shouldn't these two methods both return booleans? Then the `!!` would be unnecessary

Reply:
yep